Added Branch opcodes

luca1337 opened the pull request:
Added Branch Opcodes

Commit:
Added Branch opcodes

rdeioris commented:
Please, write a test for a branch on a page different from page 0, and you will discover that there is a bug in one of your casting.

Commit:
Fixed bad cast to unsigned char

Fixed bad unsigned char cast causing all the values over 255 limited to 255 from page 1 on, added a test for this

Commit:
Update and.c

Commit:
Changed short cast to unsigned short

Commit:
Merge branch 'luca' of https://github.com/luca1337/aiv-emulators into luca

Commit:
'Fixed' opcodes and other bugs

Commit:
Fixed branch.c

Commit:
Update branch.c

Commit:
Fixed branch tests

Commit:
Fixed branch tests

Commit:
Revert "Fixed branch tests"

This reverts commit 43c2423934c5427be3af92fe3431863673da3499.

Commit:
changed set bit

Commit:
removed json file

Commit:
fixed branch.c

Commit:
removed test

Commit:
Merge branch 'master' into luca

Commit:
Merge branch 'master' into luca
